### PR TITLE
fixed some eslint errors

### PR DESCRIPTION
**Pull request by frkozbk**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Fixed some eslint (some PropTypes, destructuring, etc.) errors.
## Related Tickets & Documents
#2470 
## Added tests?

- [ ] yes
- [X] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: app/javascript/sidebar-widget/SidebarWidget.jsx [hunk 1]
```diff
@@ -27,22 +27,24 @@ class SidebarWidget extends Component {
     });
   }
 
-  getSuggestedUsers() {
+  async getSuggestedUsers() {
```
Comment (frkozbk) [inline app/javascript/sidebar-widget/SidebarWidget.jsx]: I saw babel-polyfill inside the yarn file. So I used async/await. Should I change it to then/catch?

File: app/javascript/podcasts/PodcastFeed.jsx [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
-export const PodcastFeed = ({podcastItems}) => {
+export const PodcastFeed = ({ podcastItems }) => {
   const podcastItemDivs = podcastItems.map(ep => (
-    <a class="individual-podcast-link" href={`/${ep.podcast.slug}/${ep.slug}`}>
-      <img src={ep.podcast.image_90} /><div class="individual-podcast-link-details"><strong>{ep.title}</strong> {ep.podcast.title}</div>
+    <a
+      className="individual-podcast-link"
+      href={`/${ep.podcast.slug}/${ep.slug}`}
+    >
+      <img src={ep.podcast.image_90} alt="podcast" />
```
[the remark this answers is not in the document]
Comment (frkozbk) [reply app/javascript/podcasts/PodcastFeed.jsx]: do you mean to the 'alt' attribute?

File: app/javascript/podcasts/PodcastFeed.jsx [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
+      <h3>
+        <a href="/pod">Today&apos;s Podcasts</a>
+      </h3>
       {podcastItemDivs}
     </div>
   );
 };
 
+PodcastFeed.propTypes = {
+  podcastItems: PropTypes.arrayOf(PropTypes.object).isRequired,
```
[the remark this answers is not in the document]
Comment (frkozbk) [reply app/javascript/podcasts/PodcastFeed.jsx]: Should I remove it?

**Comment by frkozbk** [comment]
Thank you too @rhymes . I rebased this branch onto master

File: app/javascript/podcasts/PodcastFeed.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rhymes) [reply app/javascript/podcasts/PodcastFeed.jsx]: yeah sorry, but I realized we don't know what that item contains

File: app/javascript/podcasts/PodcastFeed.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (rhymes) [reply app/javascript/podcasts/PodcastFeed.jsx]: Nope, it's fine, I'm going to ask for feedback on monday. I think this component it's going to be used in the near future but I'm not sure, so for now it's fine!

**Comment by rhymes** [comment]
@fdoxyz I think something went wrong :( The branch now has 101 files and contains older changes. How did you proceed? Let me know if I can help getting it back to a correct state

**Comment by frkozbk** [comment]
@rhymes I used the script you mentioned then rebased my branch onto master

**Comment by rhymes** [comment]
Ah ok! I think the script should have been enough, sorry :( One has two choices to keep a branch up to date: either merge or rebase, but not both and each in the direction of "master -> branch", never "branch -> master".

I think the easiest solution here is to create a new branch off of the updated master, apply the two commits and create a new PR

1. `git checkout master`
1. `git checkout -b frkozbk/fix-eslint-errors-2470`
1. `git cherry-pick 94d4d4798f50acb3b3c95915538a1478ceaf9f17`
1. `git cherry-pick 81759431aabab41b7b1ab2107565085b23737beb`
1. `git push -u origin frkozbk/fix-eslint-errors-2470`

and then you open a new PR and we close this one

Again, sorry for the mishap :(

**Comment by frkozbk** [comment]
@rhymes I opened new PR. I am so sorry for the mess. Thank you for your help.

https://github.com/thepracticaldev/dev.to/pull/6521

**Comment by rhymes** [comment]
@frkozbk no mess! Sorry I wasn't clear on how to update the branch! I'll review the other one promptly!

Closed in favor of https://github.com/thepracticaldev/dev.to/pull/6521

**Pull request closed without merging**